docs/administration/configuration/configuring-workspace-retention-rules/index.md

```diff
@@ -0,0 +1,34 @@
+# Configuring workspace retention rules
```

Comment:
I would call it "Configuring global workspace retention rules" or some such as we called it in the user-facing documentation, to indicate that this is not for users, but for admins.

(Also, I think this section is perhaps too long; I'll propose some shortening it in the comments below.)